BWEnv/fbs/messages.fbs

```diff
@@ -40,7 +40,8 @@ table HandshakeServer {
   map_size:Vec2;                // walk tile x and y
   ground_height_data:[ubyte];   // walk tile resolution of ground heights
   walkable_data:[ubyte];        // walk tile resolution of walkability
-  map_name:string;
+  map_name:string;              // file name of the map (from BWAPI::Game::mapFileName())
+  map_title:string;             // in-game name of the map (from BWAPI::Game::mapName())
```

Comment:
I think if you don't add it to the end of struct (where it would be optional) you'll have to bump the version number (see "Schema evolution examples" at https://google.github.io/flatbuffers/md__schemas.html)

Reply:
Good call. I'll move it to the end.